.github/workflows/check-and-lint.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: check
+          args: --all-features
```

Comment:
As this is no longer needed, maybe remove this line?

Reply:
I saw the `all-features` flag in clippy in the same file, and thought to not remove this. But I can, no problem.

Reply:
we can also leave it as it is..